web/app/themes/mitlib-parent/functions.php

```diff
@@ -101,8 +101,12 @@ function setup_scripts_styles() {
 	wp_register_script( 'underscore', get_template_directory_uri() . '/js/libs/underscore.min.js', array(), '1.7.0', true );
 
 	// Register javascript that we've written.
-
+	wp_register_script( 'dev', get_template_directory_uri() . '/js/dev.js', array(), $theme_version, true );
+	wp_register_script( 'fonts', get_template_directory_uri() . '/js/fonts.js', array(), $theme_version, true );
```

Comment:
Not directly related to this line... by why do we have so many font files? (Not asking for a change, just a bit 🙈 )

Reply:
I don't remember right now why we have so many; it looks like the PR that added these was https://github.com/MITLibraries/MITlibraries-parent/pull/237 - neither Frances nor I discuss there why so many fonts are being added. The NTI Jira project has been archived, but those specific issues referenced don't include mention either.